src/ruby.ts

```diff
@@ -218,7 +222,7 @@ export class RubyProcess extends EventEmitter {
 
             this.debugprocess.stderr.on('data', (data: Buffer) => {
                 if (/^Fast Debugger/.test(data.toString())) {
-                    this.debugSocketClient.connect(1234);
+                    this.debugSocketClient.connect(args.debuggerPort);
```

Comment:
`args.debuggerPort` might be `undefined` so you may want to use `args.debuggerPort | 1234`, that's why the test cases fail.